packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -677,6 +679,14 @@ export class GalleryContainer extends React.Component {
     item?.offset && this.onGalleryScroll(item.offset);
   }
 
+  setScrollTop = (top) => {
```

Comment:
@IzaacAyelin 
[suggestion]
if the method isn't reused elsewhere you can remove it